### DISCUSSION TITLE
How to modify the style of child elements in Vuetify components from trame.widgets

### DISCUSSION THREAD

**Discussion by Pilot399** [Kitware/trame#452] [General]
When using Vuetify components from trame.widgets, I'm not sure how to modify the style for a specific class name within the component. For example, when using VSelect, how do I change the style of .v-select__selection?As shown, when I use VSelect, I want to change the font color and spacing of the Input field within VSelect.
![image](https://github.com/Kitware/trame/assets/158028939/f3ffdef2-babf-4068-a1c5-5030530a79d6)

**Comment by jourdain** [comment]
That might be a question for the vuetify folks, but you can provide your own css that redefine some of those style with add-on classes for the pieces you want to be tweaked.
